### DISCUSSION TITLE
Where to report this? i get scammed today on this site

### DISCUSSION THREAD

**Discussion by fuckberkleys** [community/community#74674] [Other Feature Feedback, Questions, & Ideas]
i report few times these files and nobody care and many more poeple get scammed

**Comment by Akash1134** [comment]
Hi @fuckberkleys , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- Reporting abuse or spam
- Reporting a user
- Reporting an issue or pull request
- Reporting a comment

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
